doc/tags/for.rst

```diff
@@ -66,14 +66,52 @@ Variable              Description
 ``loop.last``         True if last iteration
 ``loop.length``       The number of items in the sequence
 ``loop.parent``       The parent context
-===================== =============================================================
+``loop.cycle``        Cycle over a sequence of values
+``loop.changed``      True if previously called with a different value or if not called yet
+``loop.previous``     The value from the previous iteration (``null`` for the first iteration)
+``loop.next``         The value from the next iteration (``null`` for the first iteration)
+===================== ========================================================================
 
 .. code-block:: twig
 
     {% for user in users %}
         {{ loop.index }} - {{ user.username }}
     {% endfor %}
 
+Use ``loop.cycle`` to cycle among a list of values:
+
+.. code-block:: html+twig
+
+    {% for row in rows %}
+        <li class="{{ loop.cycle('odd', 'even') }}">{{ row }}</li>
+    {% endfor %}
+
+Use ``loop.previous`` and ``loop.next`` to compare the current value with the
+previous or next values:
```

Comment:
I would say `to access the previous or next values`, as you might use them to do something else than comparing them to the current value.

And I would also document that `loop.previous` will be `null` when being at the first item and `loop.next` will be `null` for the last item.